app/src/main/java/com/hedvig/app/feature/keygear/ui/tab/KeyGearFragment.kt

```diff
@@ -96,12 +93,12 @@ class KeyGearFragment : Fragment(R.layout.fragment_key_gear) {
             KeyGearViewModel.ViewState.Loading -> {
             }
             KeyGearViewModel.ViewState.Error -> {
-              errorContainer.root.isVisible = true
-              contentContainer.isVisible = false
+              error.show()
+              contentContainer.remove()
             }
             is KeyGearViewModel.ViewState.Success -> {
-              errorContainer.root.isVisible = false
-              contentContainer.isVisible = true
+              error.hide()
+              contentContainer.remove()
```

Comment:
Probably a `show()` here right :D
And anyway I really wonder why we even still have all the key-gear code in the codebase, I think it's pretty clear at this point we're not gonna revive it, and if we do it'd receive a complete re-design.

Reply:
I think we have it enabled for some members still. But lets check with Sara